code/modules/mod/mod_construction.dm

```diff
@@ -231,11 +231,10 @@
 				if(!user.transferItemToLoc(part, src))
 					return
 				playsound(src, 'sound/machines/click.ogg', 30, TRUE)
-				balloon_alert(user, "suit finished")
 				var/obj/item/mod = new /obj/item/mod/control(drop_location(), external_plating.theme, null, core)
 				core = null
 				qdel(src)
-				user.put_in_hands(mod)
```

Comment:
The reason i removed this is because the balloon alert gets hidden by the hands UI making it impossible to see.

Reply:
@Fikou Can you double check this? Removing this line completely seems sus.

Reply:
Surely reviewing 2 lines of code can't take 2 whole days can it?

Reply:
Ok so doing the balloon alert before putting it in hands caused the problem, but if you do it after then there is no problem